ui/analyse/src/autoShape.ts

```diff
@@ -148,7 +148,7 @@ function hiliteVariations(ctrl: AnalyseCtrl, autoShapes: DrawShape[]) {
     if (existing) {
       if (i === ctrl.fork.selected()) {
         existing.brush = brush;
-        existing.modifiers ??= {};
```

Comment:
I thought typescript was rewriting these for us already, wasn't it?

Reply:
esbuild doesn't run typescript. it just rips off the types, shits down their necks, and then calls exit(0)

Reply:
nah, that stuff may indeed have being rewritten by esbuild, but the problem was in javascipt that i was embedding directly. in client manifest.ts and also in layout.scala